examples/Microphysics/ex_1_saturation_adjustment.jl

```diff
@@ -68,7 +68,7 @@ function init_kinematic_eddy!(eddy_model, state, aux, (x, y, z), t)
     return nothing
 end
 
-function kinematic_model_nodal_update_aux!(
+function kinematic_model_nodal_update_auxiliary_state!(
```

Comment:
Could this be just `nodal_update_auxiliary_state!`? The first argument is KinematicModel.

Reply:
I don't think it would be good because this follows a pattern where the function [is passed](https://github.com/climate-machine/CLIMA/blob/master/src/Atmos/Model/AtmosModel.jl#L508-L518) to `nodal_update_aux!`, and while we might be able to use the same name, it might also be very confusing.

Reply:
I just made this change to extend the name, @trontrytel decided to append kinematic_model in front of the original function name. I decided to make these changes as minimal as possible and just update the names to make the changes I made.